riscv-sbi.adoc

```diff
@@ -1674,6 +1675,250 @@ The possible error codes returned in `sbiret.error` are shown in the
 | sbi_pmu_counter_fw_read         | 0.3         | 5   | 0x504D55
 |===
 
+== Attestation and Measurement Extension (EID #0x41545354 "ATST")
+
+[#get-evidence-function,reftext="Get Evidence"]
+=== Function: Get Evidence (FID #0)
+
+[source, C]
+----
+struct sbiret sbi_attestation_get_evidence(unsigned long cert_request_addr,
+                                           unsigned long cert_request_size
+                                           unsigned long evidence_data_addr,
+                                           unsigned long evidence_format,
+                                           unsigned long cert_addr,
+                                           unsigned long max_cert_size)
+----
+
+Retrieve a TCB attestation evidence from the underlying SBI implementation.
+Both the constitution and definition of the TCB and its measurement are
+implementation specific.
+
+The caller provides a https://datatracker.ietf.org/doc/html/rfc5280[X.509]
+formatted Certificate Signing Request (CSR) at the `cert_request_addr` physical
+address, which length is `cert_request_size`.
+
+Additionally, the caller provides a 64 bytes data blob at the
+`evidence_data_addr` physical address. This blob will be included in the
+generated certificate and can be made of any arbitrary piece of data.
+As an example, it could be a cryptographic nonce that must be bound to the
+generated evidence in order to protect the remote attestation process from
+replay attacks.
+
+The SBI implemenation replies to the signing request by generating a
+https://datatracker.ietf.org/doc/html/rfc5280[X.509] formatted certificate at
+the `cert_addr` physical address. The generated certificate must be signed with
+a platform attestation keys and must include a hash of the data blob pointed by
+`report_data_addr`. Its actual length is in `sbiret.value` and must not exceed
+`max_cert_size` bytes
+
+Moreover the generated certificate must contain:
+
+* A X.509 extension formatted according to `evidence_format`, containing the
+attestation evidence that is associated with the CSR `Subject` key. In other
+words, the generated certificate `Subject` and `SubjectPublicKey` identify the
+entity to which this extension defined evidence applies to.
```

Comment:
Avoid the dangling preposition by using something like "...identify the entity associated with this extension's defined evidence".

Reply:
Addressed. I used `In other words` as it was the beginning of the sentence.